convergedtarkus/pub_cache_cleaner
# Add basic dart setup files (pubspecs)

## ULTIMATE PROBLEM
Need basic dart setup

## HOW IT WAS FIXED
Add basic dart setup files (pubspecs)
Modified gitignore to ignore lock files and IDE files
Add comments about why files are being ignored

## TESTING AREAS:
- Make sure `pub get` correctly passes
- Make sure "unimportant" files are ignored
